modules/standard/IO.chpl

```diff
@@ -1385,8 +1388,6 @@ proc file._style:iostyle throws {
    :throws SystemError: Thrown if the file could not be closed.
  */
 proc file.close() throws {
-  try check();
-
```

Comment:
Hmm.. We'd still like the check for is_c_nil to be called. You could copy these lines from file.check and put them here:
``` chapel
  if is_c_nil(_file_internal) then
    throw SystemError.fromSyserr(EBADF, "Operation attempted on an invalid file");
```